libvast/src/format/csv.cpp

```diff
@@ -456,7 +459,7 @@ caf::error reader::read_impl(size_t max_events, size_t max_slice_size,
       continue;
     }
     if (!p(line))
-      return make_error(ec::type_clash, "unable to parse CSV line");
+      return make_error(ec::type_clash, "unable to parse CSV line", line);
```

Comment:
For JSON, we skip over invalid lines. Can we do this for CSV as well?

Reply:
Good idea, I added a status handler to match the JSON reader.